spec/controllers/v1/sessions_controller_spec.rb

```diff
@@ -237,6 +237,35 @@ def expect_logger_msg(level, msg)
           end
         end
 
+        context 'routes /sessions/custom/new to SessionController#new' do
+          it 'redirects' do
+            expect { get(:new, params: { type: :custom, authn: 'dslogon', client_id: '123123' }) }
+              .to trigger_statsd_increment(described_class::STATSD_SSO_NEW_KEY,
+                                           tags: ['context:custom', 'version:v1'], **once)
+            expect(response).to have_http_status(:found)
+            expect(response.location)
+              .to be_an_idme_saml_url('https://pint.eauth.va.gov/isam/sps/saml20idp/saml20/login?SAMLRequest=')
+              .with_relay_state('originating_request_id' => nil, 'type' => 'custom')
+              .with_params('clientId' => '123123')
+          end
+
+          it 'raises exception on missing parameter' do
```

Comment:
i think it would be nice if this test's name was explicitly saying the missing parameter is `authn`.
```
it 'raises exception when missing authn paramater' do
```

Reply:
will change